lib/phoenix/verified_routes.ex

```diff
@@ -821,7 +823,10 @@ defmodule Phoenix.VerifiedRoutes do
             "interpolated query string params must be separated by &, got: #{Macro.to_string(route)}"
     end
 
```

Comment:
You can read the environment here!

```suggestion
Application.get_env(:phoenix, :sort_verified_routes_query_params)
```

Also, let's use query_params instead of query_string since we only sort when they are rich data structures.